src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -50,10 +50,13 @@ panel {
 .tabbedPanel .tabList .tab {
 	cursor: pointer;
 	margin: auto;
-	display: flex;
 	flex-flow: row;
 }
 
+.tabbedPanel.vertical > .title .tabList .tab {
+	display: flex;
```

Comment:
There's other places that use horizonal tabs, like query results. Are you sure this is keeping all the other places behaving the same?

Reply:
I assume that's because the text isn't long enough, so I wouldn't expect there to be a change. 

@alanrenmsft, thoughts? I'm slightly concerned that someone adding in a really super long string is going to make this look terrible (and what about other languages? Did you test those @aasimkhan30?). So I'm a bit hesitant to just apply this to every single tab in the app unless we have a reason to.